README.md

```diff
@@ -1 +1,102 @@
-# -OOP-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
```

Comment:
- [ ] It would be nice if you fix the **`Install`** link because it is broken :fearful: Kindly fix it to make your readme descriptive as suggested by the previous reviewer :+1: